### PR TITLE
feat(site-header-core): Review EC, adding EU version - FRONT-1947

### PR DESCRIPTION
**Pull request by papegaill**
EC:

- preview: https://ecl-preview-1962--europa-component-library.netlify.app/playground/ec/?path=/story/components-site-headers-core--default
- specs: https://app.abstract.com/projects/2ffb1208-5760-4ed8-ab75-a3ec8e2a4ed2/branches/master/collections/a47e2bc7-65ae-4a20-b7c0-ad54a88921d9

EU:

- preview: https://ecl-preview-1962--europa-component-library.netlify.app/playground/eu/?path=/story/components-site-headers-core--default
- specs: https://app.abstract.com/projects/2ffb1208-5760-4ed8-ab75-a3ec8e2a4ed2/branches/master/collections/cd8844a3-e74e-4bd7-9a6f-1a2c9df6dbe0

### CODE REVIEW

File: src/implementations/twig/components/site-header-core/site-header-core.story.js [hunk 1]
```diff
@@ -35,17 +35,15 @@ const getArgTypes = () => {
...
-        type: { summary: 'object' },
-        defaultValue: { summary: '{}' },
-      },
-    };
-  }
+  argTypes.menu = {
+    type: { name: 'boolean' },
+    defaultValue: true,
+    description: 'Toggle menu visibility',
+    table: {
```
Comment (planctus) [inline src/implementations/twig/components/site-header-core/site-header-core.story.js]: Small thing, but i think that when the controls are nothing related to the data we should simply remove any information from the "table", it doesn't make sense to specify the data type of the control and the default value..

**Review by planctus — APPROVED** [review]
I've been just looking at the changes in the story file, basically, i leave the review of the design specs to the designers.
There is just one comment about the controls we set, but it's really a small thing.

File: src/implementations/twig/components/site-header-core/site-header-core.story.js (hunk 1 above)
Comment (papegaill) [reply src/implementations/twig/components/site-header-core/site-header-core.story.js]: Thks @planctus, PR updated.

**Review by emeryro — CHANGES_REQUESTED** [review]
there seems to be a display issue with the EC logo size on mobile (it may be very big)
image

**Comment by papegaill** [comment]
> there seems to be a display issue with the EC logo size on mobile (it may be very big)
> image

@emeryro I put back the rule for the logo in mobile format for the EC version, it should be better now.

**Review by emeryro — APPROVED** [review]

**Pull request merged**